invenio_cli/commands/assets.py

```diff
@@ -67,6 +68,37 @@ def _run_script(module_pkg):
                 status_code=status_code
             )
 
+    @staticmethod
+    def _npm_install_command(path):
+        """Run command and return a ProcessResponse."""
+        status_code = subprocess.call(["npm", "install", "--prefix", path])
```

Comment:
Interesting... Is it needed because of the `json` npm package? If not, I am fine placing it in the black-magic-of-js-packages mental bucket.